src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+
+> e.g. _ "Ethereum will take over the world because..."_
+
+**Hostile or confrontational language aimed at any organization or person**
+
+> e.g. _ "Company X is bad because they are centralized!"_
+
+**Politically charged rhetoric**
+
+> e.g. _ "This political party are better for decentralization because..."_
+
+## Tone {#tone}
+
+The tone of the content on the site should be welcoming, friendly and authoritative. Where possible, jargon should be minimized and simple language used instead.
+
+## Best Practices {#best-practices}
+
+### Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
+
+### Acroynyms {#acroynyms}
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
+
+### Mainnet {#mainnet}
+
+When referring to the Ethereum Mainnet (i.e. not referring to a testnet) use the proper noun. Proper nouns help avoid confusion and build greater understanding.
+
+**Correct usage:**
+
+- Mainnet
+- Ethereum Mainnet
+
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work {#proof-of-work}
+
+Proof-of-work should be capitalized at the beginning of a sentence or when the term is introduced for the first time on a page, in any other instance, all letters should be lower case. In either case, proof-of-work should be hyphenated between each word.
+
+**Correct usage:**
+
+- Proof-of-work
+- proof-of-work
+
+**Incorrect usage:**
+
+- Proof-of-Work
+- Proof of work
+- proof of work
+
+### Proof-of-stake {#proof-of-stake}
+
+The same rules we apply to proof-of-stake are applicable to proof-of-stake.
+
+**Correct usage:**
+
+- Proof-of-stake
+- proof-of-stake
+
+**Incorrect usage:**
+
+- Proof-of-Stake
+- Proof of stake
+- proof of stake
+
+### Use Active Voice {#use-active-voice}
+
+Sentences using active voice are more concise and efficient, making your writing more engaging and easier to comprehend.
+
+**Active voice sentence:** an actor acts on a target
+
+> _ "The man bought a car."_
+
+**Passive voice sentence:** a target acts on an actor
+
+> _ "The car was bought by a man."_
+
+[Read more on active voice](https://www.grammarly.com/blog/active-vs-passive-voice/)
+
+_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with this one._
```

Comment:
Perhaps broaden the offer for us to help?
```suggestion
_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with any of these._
```

Reply:
Good idea 👍